test/test_fx.py

```diff
@@ -641,5 +642,17 @@ def test_get_all_users_of(self):
             user_indexes = GraphManipulation.get_all_users_of(gm, i)
             assert user_indexes == expected_uses[i]
 
+    def test_find_single_partition(self):
+        class testModule(torch.nn.Module):
+            def forward(self, a, b):
+                return a + b
+        m = testModule()
+        traced = symbolic_trace(m)
+        partitioner: Partitioner = Partitioner.Partitioner()
```

Comment:
nit: If you import the Partitioner class directly this line could be:
partitioner = Partitioner() // which is easier to read.